test/dd/test_package.cpp

```diff
@@ -137,13 +138,14 @@ TEST(DDPackageTest, BellState) {
 TEST(DDPackageTest, QFTState) {
   auto dd = std::make_unique<dd::Package<>>(3);
 
-  auto h0Gate = dd->makeGateDD(dd::H_MAT, 0);
-  auto s0Gate = dd->makeGateDD(dd::S_MAT, 1_pc, 0);
-  auto t0Gate = dd->makeGateDD(dd::T_MAT, 2_pc, 0);
-  auto h1Gate = dd->makeGateDD(dd::H_MAT, 1);
-  auto s1Gate = dd->makeGateDD(dd::S_MAT, 2_pc, 1);
-  auto h2Gate = dd->makeGateDD(dd::H_MAT, 2);
-  auto swapGate = dd->makeTwoQubitGateDD(dd::SWAP_MAT, qc::Controls{}, 0, 2);
+  auto h0Gate = dd::getDD(qc::StandardOperation(0, qc::H), *dd);
+  auto s0Gate = dd::getDD(qc::StandardOperation(1_pc, 0, qc::S), *dd);
+  auto t0Gate = dd::getDD(qc::StandardOperation(2_pc, 0, qc::T), *dd);
+  auto h1Gate = dd::getDD(qc::StandardOperation(1, qc::H), *dd);
+  auto s1Gate = dd::getDD(qc::StandardOperation(2_pc, 1, qc::S), *dd);
+  auto h2Gate = dd::getDD(qc::StandardOperation(2, qc::H), *dd);
+  auto swapGate = dd->makeTwoQubitGateDD(dd::opToTwoQubitGateMatrix(qc::SWAP),
+                                         qc::Controls{}, 0, 2);
```

Comment:
You should be able to also replace this with a corresponding StandardOperation call to stay consistent. (same goes for a couple of occurences down below)